app/routes/graphkb/index.js

```diff
@@ -57,4 +63,21 @@ router.get('/statements/:statementId', async (req, res) => {
   }
 });
 
+router.post('/new-user', async (req, res) => {
+  try {
+    const {graphkbToken, body: {email, username}} = req;
+    const groupId = await graphkbGetReadonlyGroupId(graphkbToken);
+    const gkbCreateResp = await graphkbAddUser(graphkbToken, username, email, groupId);
+    if (/error/i.test(gkbCreateResp.name)) {
+      throw new Error(gkbCreateResp.message);
+    }
+    return res.status(StatusCodes.CREATED).json();
+  } catch (error) {
+    logger.error(error);
+    return res.status(StatusCodes.SERVICE_UNAVAILABLE).json({
+      message: 'GraphKB user creation error',
```

Comment:
It'd be good to include the error in the reply so the user knows why its failing, agree @elewis2 ?

Reply:
yes makes sense to me

Reply:
(pinging @kttkjl )